.pre-commit-config.yaml

```diff
@@ -5,7 +5,7 @@ repos:
       - id: end-of-file-fixer
         exclude: ^internal/(?:go|pprof)/
```

Comment:
@v-thakkar This one is making the change

```
fix end of files.........................................................Failed
- hook id: end-of-file-fixer
- exit code: 1
- files were modified by this hook

Fixing tests/testdata/java-app/src/main/resources/application.properties
```

We could move to a single `exclude` at the top-level:

https://github.com/parca-dev/demo-deployments/blob/667f077ca351965812a736a6415192ef5572ee96/.pre-commit-config.yaml#L1

Reply:
Aha, ok. Let me try doing that. And see if it works out. Thanks for the pointer!

Reply:
Yay, that worked. Thanks for unblocking me on that front. I'll take a look at why go build fails a little later in the evening.